engine-rest/engine-rest-openapi/src/main/templates/models/org/camunda/bpm/engine/rest/dto/identity/PasswordPolicyRequestDto.ftl

```diff
@@ -0,0 +1,18 @@
+<@lib.dto>
+
+    <@lib.property
+        name = "password"
+        type = "string"
+        desc = "The candidate password to be check against the password policy."
+    />
+
+    <@lib.property
+        name = "profile"
+        type = "ref"
+        dto = "UserProfileDto"
+        last = true
+        desc = "A JSON object containing variable key-value pairs. The object contains the following properties:
```

Comment:
I think this might be overly specific. It is also brittle if the properties change at some point. It is also a bit redundant, because, the dto has its own description which the user can read.